src/views/dialogs/ShareAffiliateDialog.tsx

```diff
@@ -81,13 +81,32 @@ export const ShareAffiliateDialog = ({ setIsOpen }: DialogProps<ShareAffiliateDi
         <div tw="row justify-between rounded-0.5 bg-color-layer-6 px-1 py-0.5">
           <div>
             <div tw="text-small text-color-text-0">
-              {stringGetter({ key: STRING_KEYS.AFFILIATE_LINK })}
+              {data?.isVolumeEligible
+                ? stringGetter({ key: STRING_KEYS.AFFILIATE_LINK })
+                : stringGetter({
+                    key: STRING_KEYS.AFFILIATE_LINK_REQUIREMENT,
+                    params: {
+                      AMOUNT_USD: '10K',
```

Comment:
feels like we should get this value from the backend - since the backend is the one returning the response of whether or not they are eligible due to volume.

Reply:
Yea good point, I can ask, or maybe it should be configurable from FE

Reply:
added a TODO comment for now, will follow up